test/integration-tests/shared/deployUniversalRouter.ts

```diff
@@ -55,6 +102,6 @@ export async function deployRouterAndPermit2(
   mockLooksRareToken?: string
 ): Promise<[UniversalRouter, Permit2]> {
   const permit2 = await deployPermit2()
-  const router = await deployRouter(permit2, mockLooksRareRewardsDistributor, mockLooksRareToken)
+  const router = await deployRouter(permit2, ZERO_ADDRESS, ZERO_ADDRESS, mockLooksRareRewardsDistributor, mockLooksRareToken)
```

Comment:
I think it's the wrong position for the arguments. I mean `mockLooksRareRewardsDistributor` and `mockLooksRareToken`